lib/veil/credential_collection/chef_secrets_file.rb

```diff
@@ -43,7 +45,7 @@ def initialize(opts = {})
 
         @user    = opts[:user]
         @group   = opts[:group] || @user
-        @version = opts[:version] || 1
+        opts[:version] = CURRENT_VERSION
```

Comment:
Should this be `||=`? Or do we no longer want to accept version as an arg since we detect it and don't want to write other versions out?